cli/main.c

```diff
@@ -944,29 +944,39 @@ static int log_dump(int argc, char **argv)
 	return ret;
 }
 
-#define CMD_DESC_LOG_PARSE "parse a binary app log to a text file"
+#define CMD_DESC_LOG_PARSE "parse a binary app log or mailbox log to a text file"
 
 static int log_parse(int argc, char **argv)
 {
 	int ret;
 
+	const struct argconfig_choice log_types[] = {
+		{"APP", SWITCHTEC_LOG_PARSE_TYPE_APP, "app log"},
+		{"MAILBOX", SWITCHTEC_LOG_PARSE_TYPE_MAILBOX, "mailbox log"},
+		{}
+	};
+
 	static struct {
-		struct switchtec_dev *dev;
-		int out_fd;
-		const char *out_filename;
-		unsigned type;
+		enum switchtec_log_parse_type log_type;
```

Comment:
Cleanup should happen in it's own commit. It's confusing, but I assume the fields being removed were never used in the first place?

Reply:
Right, they were never used. I'll remove them in a separate commit.